lib/changed_rpm_files_helper.rb

```diff
@@ -106,9 +106,9 @@ def get_link_target(system, link)
   def get_file_properties(system, cur_files)
     ret = {}
     out = system.run_command(
-        "stat", "--printf", "%a:%U:%G:%u:%g:%F:%n\\n",
-        *cur_files,
-        :stdout => :capture
+      "stat", "--printf", "%a:%U:%G:%u:%g:%F:%n\\n",
+      *cur_files,
+      stdout: :capture
```

Comment:
These beautifications would be better in a separate commit but also not that important.